### ISSUE TITLE
Watcher Error: "unfreezing must be generated after the unfreezing request"

### ISSUE THREAD

**Issue by spikeekips** [bosnet/sebak#897]
```
set -e
set -x

./sebak \
    node \
    --genesis GCRQV5NLUZRCU4G5QFWF2GBY4RBTENAUEGVPEECTK6QLVRBGFHGV76CV,GCPQQIX2LRX2J63C7AHWDXEMNGMZR2UI2PRN5TCSOVMEMF7BAUADMKH5,5000000000000000 \
    --network-id "sebak-network; 2018-11-26" \
    --storage "file://$(pwd)/db" \
    --bind "http://0.0.0.0:12345" \
    --log-level debug \
    --secret-seed SDKM7MD5VSVFRE4NXHPWBZGCOC2RVWJHP4UMB7ZMHL6WLLBVEGOZL45C \
    --validators "self" \
    --rate-limit-api "100-S" \
    --http-log ./http.log \
    --discovery https://mainnet.blockchainos.org \
    --watcher-mode \
    --set-congress-address
    GDK4CSKXCXKJB5LOQLJO55WIUGMIRGYGRKJ5RV7FLLR2D3GVSPIE3S7B \
    $* 2>&1 &> ./node.log
```
This node is,
* version is `v0.1.3`
* as you see, watcher mode
* watching target is mainnet(https://mainnet.blockchainos.org)

The syncing and watching was running well, but it occurred this error and stuck.
```log
t=2019-01-03T22:49:25+0000 lvl=dbug msg="start validate txs" module=sync node=GBBH.KHKY submodule=validator height=291146 caller=validator.go:189
t=2019-01-03T22:49:25+0000 lvl=eror msg="validate failure" module=sync node=GBBH.KHKY submodule=syncer err="{\"code\":170,\"message\":\"unfreezing must be generated after the unfreezing request\",\"data\":{}}" height=291146 caller=syncer.go:266
```
The error message is,
```json
{
  "code": 170,
  "message": "unfreezing must be generated after the unfreezing request",
  "data": {}
}
```

The latest node info is,
```json
$ curl localhost:12345
{
  "node": {
    "version": {
      "version": "0.1.3",
      "git-commit": "83efb8e7",
      "git-state": "clean",
      "build-date": "2018-12-27T03:49:54Z"
    },
    "started": "2019-01-02 22:33:17.352880458 +0000 UTC m=+0.007074933",
    "state": "WATCH",
    "alias": "GBBH.KHKY",
    "address": "GBBHVSKG2MAIOQAX3UHOVUJXYLHTZGBKXY5AR2AVD54EEPH5KHKYQIAQ",
    "endpoint": "http://localhost:12345",
    "validators": {
      "GBBHVSKG2MAIOQAX3UHOVUJXYLHTZGBKXY5AR2AVD54EEPH5KHKYQIAQ": {
        "address": "GBBHVSKG2MAIOQAX3UHOVUJXYLHTZGBKXY5AR2AVD54EEPH5KHKYQIAQ",
        "alias": "GBBH.KHKY",
        "endpoint": "http://localhost:12345"
      }
    }
  },
  "policy": {
    "network-id": "sebak-network; 2018-11-26",
    "initial-balance": "5000000000000000",
    "base-reserve": "1000000",
    "base-fee": "10000",
    "block-time": 5000000000,
    "block-time-delta": 1000000000,
    "timeout-init": 2000000000,
    "timeout-sign": 2000000000,
    "timeout-accept": 2000000000,
    "timeout-allconfirm": 30000000000,
    "rate-limit-api": "100-S",
    "rate-limit-node": "100-S",
    "transactions-limit": 1000,
    "operations-limit": 1000,
    "operations-in-ballot-limit": 10000,
    "genesis-block-confirmed-time": "2018-04-17T5:07:31.000000000Z",
    "inflation-ratio": "0.00000010000000000",
    "unfreezing-period": 241920,
    "block-height-end-of-inflation": 36000000
  },
  "block": {
    "height": 291145,
    "hash": "5juiEw1mMMAV37AWEdLuAR6DFov4ypZr9H2JQwEngBUK",
    "total-txs": 295856,
    "total-ops": 600858,
    "proposed": "2018-12-14T05:26:25.940843095Z",
    "confirmed": "2018-12-14T05:26:25.953699266Z"
  }
}
```

**Comment by anarcher** [comment]
> node_1  | t=2019-01-04T04:45:40+0000 lvl=info msg="done sync work" module=sync node=GBVG.MAIO submodule=syncer height=424844 hash=G7iexnLf95juzre6nUje55WHXwnASJBcjKWri63CeMqG

With master branch, This height(291146)  is synced. :->

**Comment by spikeekips** [comment]
@anarcher I will test with the master branch.

**Comment by soonkuk** [comment]
@spikeekips @anarcher I think this error was also generated from the reason operation saved lately.

**Comment by soonkuk** [comment]
To validate unfreezing payment operation, ValidateOp checks previous unfreezing request operation.
If unfreezing request operation is not saved yet, it may make that error.

**Comment by soonkuk** [comment]
I tested watcher mode repeatedly with version 0.1.4 in MainNet. But I did not find above error.
I think this issue was resolved by #893
What do you think about closing this issue?
